goto-line-preview.el

```diff
@@ -87,9 +87,11 @@
         jumped)
     (run-hooks 'goto-line-preview-before-hook)
     (unwind-protect
-        (setq jumped (read-number (if goto-line-preview--relative-p
-                                      "Goto line relative: "
-                                    "Goto line: ")))
+        (setq jumped (read-number
+		      (let ((lines (count-lines (point-min) (point-max))))
```

Comment:
Why not use `line-number-at-pos` with `(point-max)`? Just curious.

Reply:
@jcs090218 Thanks for your revirw, actually I'm not very familiar with elisp, if `(line-number-at-pos (point-max))` is better, I can edit the PR.

Reply:
How about?

```elisp
(let ((lines (line-number-at-pos (point-max))))
  (format (if goto-line-preview--relative-p
              "[%d] Goto line relative: (%d to %d) "
            "[%d] Goto line: (%d to %d) ")
          goto-line-preview--prev-line-num
          (max 0 (min 1 lines))
          lines))
```

I've changed the format a little and added the current line in the front. Example output:

```
[13] Goto line: (1 to 100) 
```

P.S. It seems like there are indention error there, can you fix that for us?

Reply:
@jcs090218 Thanks for your advise, I fix the code, would please review again?